platforms/minigame/platforms/taobao/wrapper/builtin/WebSocket.js

```diff
@@ -1,7 +1,7 @@
 const _utils = require('../utils');
 
-const MAX_VALUE_WEBSOCkET = 1;
-let CURR_VALUE_WEBSOCKET = 0;
+const MAX_REF_WEBSOCKET = 1  // The maximum number of WEBSOCKET
```

Comment:
Why change to ref?

Reply:
The word `VALUE` is vague, Maybe the word `NUM` is better than the word `REF`.